src/extensions/default/HTMLCodeHints/main.js

```diff
@@ -319,10 +318,9 @@ define(function (require, exports, module) {
                 
                 // If we're at an attribute value, check if it's an attribute name that has hintable values.
                 if (this.tagInfo.attr.name) {
-                    var hintsAndSortFunc = this._getValueHintsForAttr({queryStr: query},
-                                                                      this.tagInfo.tagName,
-                                                                      this.tagInfo.attr.name);
-                    var hints = hintsAndSortFunc.hints;
+                    var hints = this._getValueHintsForAttr({queryStr: query},
```

Comment:
I'm not sure I understand what you want here.  You want me to break alignment like this?

```js
                    var hints = this._getValueHintsForAttr({queryStr: query},
                                                                      this.tagInfo.tagName,
                                                                      this.tagInfo.attr.name);
```